devtools-ci-index.yaml

```diff
@@ -3746,6 +3746,28 @@
             git_repo: fabric8-analytics-npm-insights
         - '{ci_project}-{git_repo}-fabric8-analytics-pydoc':
             git_repo: fabric8-analytics-npm-insights
+        - '{ci_project}-{git_repo}-fabric8-analytics':
+            git_organization: fabric8-analytics
+            git_repo: f8a-golang-insights
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_run_tests.sh'
+            timeout: '20m'
+            registry: 'quay.io'
+            image_name: 'openshiftio/rhel-fabric8-analytics-f8a-golang-insights'
```

Comment:
IIRC this is the image name that will be reported back to GitHub, for PR builds. People cannot pull RHEL-based images so I might be better to use CentOS-based image name here.

Reply:
I actually thought of doing that but I saw that everywhere else in our templates we had this set to RHEL so did that here too. But now that I know for sure what this is used for I'll correct it, thanks! 😄

Reply:
Done, can you test it again for me please? 🙂